bin/improver-weighted-blending

```diff
@@ -123,13 +128,24 @@ def main():
 
     cube = iris.load_cube(args.input_filepath)
 
+    if args.coord_exp_val:
```

Comment:
Changed to use list comprehension. Calculation of coordinated has been moved into process_coord.